app/assets/stylesheets/layout/_content.scss

```diff
@@ -22,30 +22,6 @@
   }
 }
 
-.performance-reports.show {
-  h2 + h3 {
-    margin-top: 30px;
-  }
-  .table {
-    margin-top: 15px;
-    margin-bottom: 30px;
-
-    td:nth-child(2) {
-      width: 90px;
-      text-align: right;
-      padding-right: 20px;
-    }
-
-    .active {
-      td {
-        font-weight: bold;
-        border-bottom: 0;
-      }
-
-      & + tr td {
-        border-top: 0;
-        padding-top: 25px;
-      }
-    }
-  }
+.row {
+  margin-bottom: 10px;
```

Comment:
I just untangled and reorganized the whole css. It was globaly effective before. So no change there...

Reply:
https://github.com/panter/aoz-003/pull/196/files#diff-84f8fd7003c0c9744039eb89c5f85453L20

Reply:
https://github.com/panter/aoz-003/blob/85596a8bc3ab773d02e929c5d505ca2412d9a9e7/app/assets/stylesheets/layout/_pdf.scss#L20

Reply:
It may have been a mistake back then, but as it was globally applied there, I would change everything by removeing it now. So thats why I've put it there.

Reply:
Ah ok nevermind, didn't catch that :)